packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -22,273 +22,330 @@ related: [
 
-#### Top-aligned labels
+## User guidance
 
-<img src="./img/top-labels.png" alt="Example of label aligned to the top of the input box" width="444"/>
+Due to their interactive nature, forms often include features that help communicate field importance and guidance as users complete them. 
 
-Use top-aligned labels by default. Top-aligned labels provide a consistent left edge and close proximity between label and input. Top-aligned labels also offer the most horizontal area for long labels.
+Types of user guidance include:
 
-#### Left-aligned labels
+* **Required fields:** Call attention to input fields crucial to user success.
+* **User help:** Assists users in context through placeholder text, helper text, and popovers.
+* **Progressive disclosure:** Hides and shows fields based on user selections to simplify their workflow.
 
-<img src="./img/left-labels.png" alt="Example of label aligned to the left of the input box" width="444"/>
+### Required fields
 
+Reserve required fields for information vital to user success. Indicate a required field with an asterisk ( * ) to the right of its field label.
 
-Left-aligned labels are another available form option. They are useful for layouts with more horizontal space than vertical. However, as a general rule, they should be avoided in favor of top-aligned labels.
+<img src="./img/required-field.png" alt="Example of field with required field indication" width="444"/>
 
-To accommodate mobile viewports, left-aligned labels should responsively adjust to a top-aligned layout for smaller viewports.
+If all form fields are required, don’t use an asterisk for each one. Instead, provide a message at the top of the form: “All fields are required.” If all fields are optional, provide a similar message at the top of the form: “All fields are optional." 
 
-#### Checkbox and radio button labels
-Checkboxes and radio buttons have a different labeling convention, where the option labels should be aligned to the right of the input control. 
+<img src="./img/required-optional-forms.png" alt="Example of two forms with an indication at the top of whether all fields are required or all fields are optional"/>
 
-<img src="./img/radio-check.png" alt="Example of checkboxes and radio buttons aligned to the right of the input control" width="376"/>
 
-#### General labeling guidelines
-* Use sentence-style capitalization for all text elements except: product names, proper nouns and acronyms. Sentence-style capitalizes only the first word of each sentence and proper nouns, like names.
-* Always provide a label for input components.
-* Labels should clearly state the required input.
-* **Do not** use a colon after the label name.
+### User help
 
-### Required fields
-A required field should be indicated with an asterisk ( * ) to the right of the field label.
+There are three different methods for providing assistance within a form: Placeholder text, helper text, and popovers.
```

Comment:
Would link "placeholder text" "helper text" and "popovers" to their place in the doc! 

So example: [placeholder text](#placeholder-text)